com.reprezen.swagedit.dependencies/pom.xml

```diff
@@ -53,22 +78,71 @@
 						<goals>
 							<goal>copy-dependencies</goal>
 						</goals>
+						<configuration>
+							<outputDirectory>${project.basedir}/lib</outputDirectory>
+							<overWriteReleases>false</overWriteReleases>
+							<overWriteSnapshots>true</overWriteSnapshots>
+							<includeTypes>jar</includeTypes>
+							<prependGroupId>true</prependGroupId>
+							<stripVersion>true</stripVersion>
+							<excludeArtifactIds>com.reprezen.swagedit.dependencies</excludeArtifactIds>
+							<excludeGroupIds>
+								com.google.guava,
+								javax.mail,
+								javax.activation,
+								com.sun.mail
+							</excludeGroupIds>
+							<excludeScope>system</excludeScope>
+							<includeScope>runtime</includeScope>
+						</configuration>
+					</execution>
+					<execution>
+						<id>classpath</id>
+						<goals>
+							<goal>build-classpath</goal>
+						</goals>
+						<configuration>
+							<prefix>lib</prefix>
+							<prependGroupId>true</prependGroupId>
+							<outputFile>${project.basedir}/lib/classPath.txt</outputFile>
```

Comment:
It looks to me like the purpose of this configuration is to create the `${project.basedir}/lib/classPath.txt`  file, but I don't see that filename referenced anywhere else. Is it used by one of our build scripts on TC?

Reply:
Because it's produced by maven during project build.

Reply:
OK, I think the explanation is here, in the same commit comment you referenced above:

> The maven build will also add a file named classPath.txt into the lib folder. This is done by the maven dependency plugin. This file will contain dependency versions that are needed by the gentemplate wizard to set versions of gentemplate projects dependencies (such as genflow, kzop, rapidml).

So the Maven build produces it, and it's used at runtime by the GenTemplate wizard.